### PR TITLE
Update spanish locs

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: webapp/i18n/es.json [hunk 1]
```diff
@@ -378,7 +378,8 @@
   "admin.reset_password.newPassword": "Nueva contraseña",
   "admin.reset_password.select": "Seleccionar",
   "admin.reset_password.submit": "Por favor, introducir como mínimo 5 caracteres.",
-  "admin.reset_password.title": "Reiniciar Contraseña",
+  "admin.reset_password.titleReset": "Restablecer la contraseña",
+  "admin.reset_password.titleSwitch": "Cambiar cuenta a Correo Electrónico/Constraseña",
```
Comment (Strangehill) [inline webapp/i18n/es.json]: Constraseña --> Contraseña

File: webapp/i18n/es.json [hunk 2]
```diff
@@ -563,6 +564,7 @@
   "admin.user_item.mfaYes": ", <strong>AMF</strong>: Sí",
   "admin.user_item.resetMfa": "Remover AMF",
   "admin.user_item.resetPwd": "Reiniciar Contraseña",
+  "admin.user_item.switchToEmail": "Cambiar a Correo Electrónico/Constraseña",
```
Comment (Strangehill) [inline webapp/i18n/es.json]: Constraseña --> Contraseña

Comment (enahum) [inline webapp/i18n/es.json]: Gracias @Strangehill

**Comment by enahum** [comment]
@Strangehill are you a native spanish speaker? would you like to join [Localization Channel](https://pre-release.mattermost.com/core/channels/localization) and help us review translations?

Thanks.

**Comment by Strangehill** [comment]
I am, and sure! 
I've joined the localization channel by following the link.
I'm unsure of the next steps to take though.

**Pull request merged**